lua/wikis/commons/TeamParticipants/Parse/Wiki.lua

```diff
@@ -34,6 +41,50 @@ function TeamParticipantsWikiParser.parseWikiInput(args)
 	}
 end
 
+---@param input table?
+---@return QualificationStructure?
+local function parseQualifier(input)
+	if not input then
+		return
+	end
+	local qualificationMethod = input.method
+	if not qualificationMethod then
+		return
+	end
+
+	local qualificationType
+	if input.page then
+		qualificationType = 'tournament'
+	elseif input.url then
+		qualificationType = 'external'
+	else
+		qualificationType = 'other'
+	end
+
+	local qualificationStructure = {
+		method = qualificationMethod,
+		type = qualificationType,
+		text = input.text
+	}
+
+	if qualificationType == 'tournament' then
+		local tournament = Tournament.getTournament(input.page)
+		if not tournament then
+			qualificationStructure.type = 'other'
+		else
+			qualificationStructure.tournament = tournament
+		end
+	elseif qualificationType == 'external' then
+		qualificationStructure.url = input.url
+	end
+
+	if qualificationType == 'external' and not qualificationStructure.text then
```

Comment:
Wouldn't `other` also be required to have text?
Especially in the case when that's auto-assigned based on a page that's not a tournament

Edit: Probably not, other will just display the method (+text if set) and ignore the page

Reply:
Indeed, on your edito